### PR TITLE
Move previous/next versions pane.

### LINKED ISSUE #1444: Move "Previous version" and "Next version" buttons down
Just noticed these buttons -

I think they're too far up on the page - they're not as important as the info of the gem, authors, etc. I think they must be useful for some folks - but maybe below the checksum would be a better spot.

Now:

before

Perhaps this?:

after

### PR DESCRIPTION
**Pull request by harrykiselev**
Hello, rubygems! :)

I want to resolve #1444 issue.

### CODE REVIEW

**Comment by sonalkr132** [comment]
Hi @harrykiselev :wave: 
Thanks for PR. Can you please post a screenshot?

**Comment by harrykiselev** [comment]
@sonalkr132 page of the screenshot is infomation about 1.2.2 version. "Next version" is hovered :)

File: test/integration/gems_test.rb [hunk 1]
```diff
@@ -71,9 +71,9 @@ class GemsSystemTest < SystemTest
 
   test "version navigation" do
     visit rubygem_version_path(@rubygem, "1.0.0")
-    click_link "Next version"
+    click_link "Next version →"
```
Comment (simi) [inline test/integration/gems_test.rb]: :question:

**Comment by simi** [comment]
Big up for this. :+1:

File: test/integration/gems_test.rb (hunk 1 above)
Comment (harrykiselev) [inline test/integration/gems_test.rb]: I thought it's correct. This page links has a new text with arrows. Though tests passed without arrows. I'd better remove it? :)

**Pull request merged**